src/components/Map/AreaHighlightMap.tsx

```diff
@@ -121,16 +121,20 @@ const AreaLocationMap = ({ regions, id }: AreaLocationMapProps) => {
         zoom={Object.keys(addresses).length === 1 ? 8 : 3}
         center={getDefaultCenter(addresses)}
       >
-        {data.markerPoint.map(([x, y], index) => (
-          <Marker
-            key={x}
-            position={new naver.maps.LatLng(y, x)}
-            onClick={() => selectMarker(new naver.maps.LatLng(y, x))}
-            icon={{
-              content: `<div class="markerBox">${markerName[index]}</div>`,
-            }}
-          />
-        ))}
+        {data.markerPoint.map(([x, y], index) => {
+          if (!markerName[index]) return;
```

Comment:
```suggestion
          if (!markerName[index]) return null;
```
여기 렌더링 부분이라 그냥 return하게 되면 오류 발생할 수도 있어요ㅠㅅㅠ

Reply:
그러게 null 빼묵었네 땡큐 👍